tests/nested_test.cpp

```diff
@@ -318,6 +318,18 @@ void test_exception_propagating_from_nested_generator() {
     CHECK(it == g.end());
 }
 
+void test_elementsof_with_allocator_args() {
+    std::vector<int> v;
+    auto with_alloc = [&] (std::allocator_arg_t, std::allocator<std::byte> a) -> std::generator<int> {
+            co_yield 42;
+            //co_yield std::ranges::elements_of(v);
+            co_yield std::ranges::elements_of(v, a);
+    };
```

Comment:
I think you also need a test that tests that you can pass an allocator to elements_of even if the parent coroutine does not have an explicit allocator.